lib/aur-build

```diff
@@ -122,8 +126,7 @@ while true; do
             shift; pacman_conf=$1
             chroot_args+=(--pacman-conf "$1") ;;
         --pkgver)
-            run_pkgver=1; makepkg_args+=(--noextract)
-            chroot_build_args+=(--margs '--holdver') ;;
+            run_pkgver=1; makepkg_args+=(--noextract) ;;
```

Comment:
I thought you didn't particularly like the `--margs` interface with the comma as separator and it felt like a good moment for breaking changes, more so because there is still --create/--update to add more packages to the environment.
Just put a thumbs up and I'll do what you asked, no biggie.

Reply:
The challenge is when combining `--build` with `--create` / `--update`. Then the additional arguments could be both `makepkg` arguments, and package names. With the end-of-options approach, to pass on additional packages to be installed in the chroot you'd need an additional flag like `--extra-packages`. Since that approach gives some special meaning for `--` when `--margs` doesn't, I think `--margs` is preferable overall.